ToDoGeo/ToDoGeo/System/SceneDelegate.swift

```diff
@@ -6,17 +6,28 @@
 //
 
 import UIKit
+import class RxSwift.DisposeBag
+import RxFlow
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
-
+    var coordinator = FlowCoordinator()
+    var disposeBag: DisposeBag = DisposeBag()
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
-        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
-        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+
+        guard let windowScene = (scene as? UIWindowScene) else { return }
```

Comment:
가독성을 위해 아래와 같이 변경 부탁드립니다. 

```suggestion
        guard let windowScene = (scene as? UIWindowScene) else { 
            return 
        }
```

Reply:
[e93bfb1] 수정한 커밋입니다.